code/modules/mob/living/simple_animal/abnormality/he/funeral.dm

```diff
@@ -11,6 +11,7 @@
 
 	ranged = TRUE
 	minimum_distance = 2
+	retreat_distance = 1
```

Comment:
Shouldnt it be the same as minimum distance or else it will walk to minimum and retreat wonkily?